src/types.ts

```diff
@@ -2268,6 +2268,16 @@ export enum HostEvent {
      * @hidden
      */
     Reload = 'reload',
+    /**
+     * Get current iframe src
+     * @example
+     * ```js
+     * const frameUrl = AppEmbed.trigger(HostEvent.GetIframeUrl)
+     * ```
+     * @version SDK: 1.35.0 | Thoughtspot: 10.4.cl
+     * @hidden
```

Comment:
Why @hidden tag is added?

Reply:
Removed, It's not available in docs, will raise one ticket for @ShashiSubramanya